integration/acceptance/splitter_spec.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
```

Comment:
don't think this is necessary anymore? Strings are using the UTF-8 encoding by default.

Reply:
Good point - this application is effectively Ruby 2 only, so these aren't necessary. I will take them out.